.github/workflows/twister.yaml

```
@@ -210,6 +210,10 @@ jobs:
           echo "github.base_ref: ${{ github.base_ref }}"
           echo "github.ref_name: ${{ github.ref_name }}"
 
+      - name: Install Python packages
+        run: |
+          pip install -r scripts/requirements-actions.txt --require-hashes
```

Need to setup Python too?

```suggestion
    - name: Set up Python
      uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5.6.0
      with:
        python-version: 3.12
        cache: pip
        cache-dependency-path: scripts/requirements-actions.txt

    - name: Install Python packages
      run: |
        pip install -r scripts/requirements-actions.txt --require-hashes

```

---

I dunno, I guess -- I never understand how this setup step is supposed to work (esp. since here we already have python in the container)

---

> Need to setup Python too?

No, we should be using the Python included in the CI Docker image. If we need to change the Python version, the Docker image should be updated instead.